.github/workflows/docker-sandbox.yml

```diff
@@ -29,25 +29,25 @@ jobs:
         username: ${{ env.DOCKER_USER }}
         password: "${{ secrets.DockerPassword }}"
         push_image_and_stages: false
```

Comment:
so it won't upload caches?

Reply:
Yep.

I don't think you can only push caches... and you wouldn't want it to.